.github/workflows/test_openvino_slow.yml

```
@@ -65,7 +58,7 @@ jobs:
 
       - name: Install dependencies (slow)
         run: |
-          pip install ${{ matrix.nncf }}
+          pip install .[nncf]
```

not necessary for now since nncf doesn't constrain transformers but just in case it did
```suggestion
          pip install .[nncf]

      - if: ${{ matrix.transformers-version != 'latest' }}
        name: Downgrade Transformers and Accelerate
        run: pip install transformers==${{ matrix.transformers-version }} accelerate==0.*
```